### PR TITLE
Add styled components and remove sass

### LINKED ISSUE #24: Use `styled-components` as a primary way of styling React components

### PR DESCRIPTION
**Pull request by pziemkowski**
Closes #24

### CODE REVIEW

File: app/theme/spriteImage.component.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const mobileSprite = (props) => {
+  const name = props.mobile || props.name;
+  if (name) {
+    return mobile(name);
+  }
+
+  return null;
+};
+
+const desktopSprite = (props) => {
```
Comment (mkleszcz) [inline app/theme/spriteImage.component.js:15]: What do  you think about add also tablet sprite here?

File: app/routes/home/home.container.js [hunk 2]
```diff
@@ -18,4 +19,4 @@ export const mapDispatchToProps = (dispatch) => bindActionCreators({
   setLanguage: LocalesActions.setLanguage,
 }, dispatch);
 
-export default connect(mapStateToProps, mapDispatchToProps)(Home);
+export default hot(module)(connect(mapStateToProps, mapDispatchToProps)(Home));
```
Comment (mkleszcz) [inline app/routes/home/home.container.js:22]: So now we'll need to add this HOC to every component/container which we would like to hot reload?

Comment (pziemkowski) [reply app/routes/home/home.container.js:22]: In every container is enough. We'll add it to the plop generator.

File: app/theme/spriteImage.component.js (hunk 1 above)
Comment (pziemkowski) [reply app/theme/spriteImage.component.js:15]: Well, I've removed it, since not all projects need tablet sprites :)

**Review by mkleszcz — APPROVED** [review]

**Pull request merged**